.github/workflows/ci.yml

```diff
@@ -38,3 +38,14 @@ jobs:
         run: cargo build
       - name: Run tests
         run: cargo test -v
+
+  msrv:
+    name: Test using MSRV
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - run: rustup toolchain install $(awk '/^rust-version/ { gsub(/"/, ""); print $3 }' Cargo.toml) --profile minimal --no-self-update
+      - run: cargo build --all-targets
+      - run: git submodule update --init
+      - run: cargo test --all-targets
```

Comment:
Can this PR be rebased over `master` and the following changes be made:
* I'd like to ensure MSRV can go down to 1.65.0 with all features disabled. Would it be possible to have an extra key (even if it's commented) in `Cargo.toml`, like `rust-version-no-features` that would contain the `1.65.0`? This is a temporary ugly hack.
* Can you run `cargo check` instead of compiling and running all tests? This also removes the `git submodule update`.

The defaults for `yaml-rust2` are wrong on many levels and fixing them is a task for future me.

Reply:
Will look in the next few days.